HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -1457,6 +1473,7 @@ def self.process_equipment_adjustments(hvac_final_values, weather, hvac)
                                    (1.0 + (1.0 - hvac.CoilBF * bypassFactor_CurveValue) *
                                    (80.0 - @cool_setpoint) / (@cool_setpoint - hvac.LeavingAirTemp)))
         hvac_final_values.Cool_Airflow = calc_airflow_rate(cool_Load_SensCap_Design, (@cool_setpoint - hvac.LeavingAirTemp))
+        # FIXME: the airflow was adjusted to about 1.7 * airflow_preadjusted, please verify it.
```

Comment:
@yzhou601 Does this still need to be done?

Reply:
No it's now reasonable with bugfix.